File: lib_common/src/main/java/com/ebook/common/BaseApplication.java
package com.ebook.common;

import android.app.Application;

import com.ebook.common.util.log.KLog;
import com.facebook.stetho.Stetho;

public class BaseApplication extends Application {
    private static BaseApplication mApplication;

    public static BaseApplication getInstance() {
        return mApplication;
    }

    @Override
    public void onCreate() {
        super.onCreate();
        mApplication = this;
        KLog.init(BuildConfig.IS_DEBUG);
        Stetho.initializeWithDefaults(this);
    }
}

File: lib_common/src/main/java/com/xrn1997/common/BaseApplication.kt
package com.xrn1997.common

import android.annotation.SuppressLint
import android.app.Application
import android.content.Context
import com.facebook.stetho.Stetho


/**
 * BaseApplication
 */
open class BaseApplication : Application() {
    companion object {
        @SuppressLint("StaticFieldLeak")
        @JvmStatic
        lateinit var context: Context
    }

    override fun onCreate() {
        super.onCreate()
        context = applicationContext
        Stetho.initializeWithDefaults(this)
    }


}
